get_started/what_is_citus.rst

```diff
@@ -44,13 +43,13 @@ Citus scales horizontally by adding worker nodes, and vertically by upgrading wo
 * `Pex <https://www.citusdata.com/customers/pex>`_
 
   * 80B rows updated/day
-  * 20-node Citus database cluster on Google Cloud
+  * 20-node Citus database cluster
   * 2.4TB memory, 1280 cores, and 80TB of data
   * ...with plans to grow to 45 nodes
 
 * `MixRank <https://www.citusdata.com/customers/mixrank>`_
 
-  * 10+ PB of time series data
+  * 1.6 PB of time series data
```

Comment:
Oh, did this change back again? I recall increasing the number recently.

Reply:
you're right, it's 10PB. fixed.